README.md

```diff
@@ -46,4 +46,16 @@ $ pip install -r requirements.txt
    $ source venv/bin/activate
    $ python3 tools/format_target.py [原文フォルダのパス(デフォルト: source)] [訳文フォルダのパス(デフォルト: target)]
    ```
-2. `source` フォルダと `target` フォルダを比較して、有意な変更のあった訳文ファイルのみ、Transifexにアップロード
\ No newline at end of file
+2. `source` フォルダと `target` フォルダを比較して、有意な変更のあった訳文ファイルのみ、Transifexにアップロード
+
+
+## 訳文の生成
+
+
+```
+
+$ ./gradlew trasnlate
```

Comment:
@miurahr I think that this is typo of `translate`.

Reply:
Also, before/after empty line may not be necessary.
And adding `bash` may be good for GitHub Markdown code style.